src/config.ts

```diff
@@ -46,11 +45,9 @@ const variablesToCheck = [
   ...(process.env.NODE_ENV === 'production' ? mandatoryVariablesOnlyInProd : []),
 ];
 
-
-
 const undefinedMandatoryVariables = variablesToCheck.filter((key) => !process.env[key]);
 
-if (undefinedMandatoryVariables.length) {
+if (!isTestEnv && undefinedMandatoryVariables.length) {
```

Comment:
The test environment will still require the environment variables to be present. This seems like a bad idea.